src/service/auth/login.ts

```
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express';
+import { prisma } from '../../config/prisma';
+import bcrypt from 'bcrypt';
+import redis from '../../config/redis';
+import { LoginResponse, SignRequest } from '../../types/auth';
+import { generateToken } from './token';
+
+export const login = async (req: Request<{}, {}, SignRequest>, res: Response<LoginResponse | object>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)